docs/en/ingest-management/elastic-agent/running-on-kubernetes-gke-standard-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,170 @@
+[[running-on-kubernetes-gke-standard-managed-by-fleet]]
+= Run {agent} on GKE standard managed by {fleet}
```

Comment:
should this documentation be more an `Observe GKE standard with elastic-agent managed by fleet`, then the run fleet?

Reply:
So also the title should not have kubenretes word. I think is duplication
[running-on-kubernetes-gke-standard-managed-by-fleet.asciidoc] -> [running-on-gke-standard-managed-by-fleet.asciidoc]

Reply:
I like the idea of making the titles more user goal oriented. I wonder what users are likely to search for when they go to google or our docs?